docs/schema.md

```diff
@@ -696,13 +707,15 @@ The defined ecosystems are:
 | `npm` | The NPM ecosystem; the `name` field is an NPM package name.  |
 | `NuGet` | The NuGet package ecosystem. The `name` field is a NuGet package name.  |
 | `OSS-Fuzz` | For reports from the OSS-Fuzz project that have no more appropriate ecosystem; the `name` field is the name assigned by the OSS-Fuzz project, as recorded in the submitted fuzzing configuration.  |
+| `openSUSE` | The openSUSE ecosystem; The ecosystem specifier is the marketing name of the openSUSE distribution. With the `:` removed, it matches the value in the `/etc/os-release` `PRETTY_NAME` field. The `name` field is the name of the source RPM and accompanied by a purl. There is an `ecosystem_specific` specific array `binaries` of the associated RPM binary packages in this specific openSUSE distribution. The ECOSYSTEM version ordering is the RPM versioncompare ordering, and the database uses the `introduced` and `fixed` boundaries.|
```

Comment:
Would you be able to make this consistent with the text for `SUSE` ? i.e. "The ecosystem string has a `:<RELEASE>` suffix representing the marketing name of the openSUSE distribution..."

Reply:
done so